blogposts/2022/third-party-vulnerabilties-process.md

```diff
@@ -0,0 +1,62 @@
+
+There’s also a cultural shift you can make: making fixing vulnerabilities an engineering-wide effort instead of the burden of a small team or lone security engineer can exponentially speed up your response time, as in addition to more people tackling the problem, the developers themselves may be able to work faster as they’re more familiar with the parts of the codebase they’re investigating.
+
+## 3. Mitigation is too time consuming
+
+If you _do_ have exploitable code, you’ll need to upgrade to the next safe version as soon as possible. If doing so is going to introduce breaking changes though, you may need more fine-grained mitigation that refactors the vulnerable code, wherever it’s being used. In either case, this can be a tedious, manual process involving dozens of pull requests across multiple repositories.
+
+### How to fix it
+
+The cultural shift of making remediation everyone’s responsibility can drastically speed up your response time here too. You can also programmatically create GitHub pull requests or GitLab merge requests with your mitigations using Sourcegraph Batch Changes, reducing the manual effort of creating and applying individual fixes. You can see [how this worked for Log4j mitigation here](https://about.sourcegraph.com/blog/log4j-log4shell-0-day/#Automate-PRs-to-fixmitigate-the-log4j-0-day-across-all-your-code).
+
+## 4. You’re not prepared for future vulnerabilities
+
+Vulnerabilities in third-party code are inevitable, but stripping your codebase of open source software isn’t the answer, unless you want to stifle innovation and frustrate all your developers. The challenge is in keeping dependencies manageable and up to date and ensuring consistency in your use of shared libraries. If left unchecked, you’re in for [dependency hell](https://about.sourcegraph.com/blog/nine-circles-of-dependency-hell/): package bloat, circular dependencies, and reckless monkey-patching are just some of the symptoms.
+
+### How to fix it
+
+You can proactively reduce your dependencies by committing to a culture of code reuse in your engineering organization. Code reuse doesn’t just narrow the scope of your dependencies, but reduces time spent on writing duplicative code and streamlines maintenance of shared code. Code search can help engineers discover reliable, reusable code already in your codebase. Using code notebooks to document how and where code is used in your organization also makes it easier for developers to reuse shared libraries confidently, reducing your overall dependency burden.
+
+_TODO: Add summary with link to fixing vulnerabilities with Nutanix webinar when landing page is live_
```

Comment:
TODO

Reply:
@jennybergen I think the answer is no, but do you know if we have something to link to here yet?

Reply:
I will come back and add this when we have it